ci-operator/config/openshift/lvm-operator/openshift-lvm-operator-main.yaml

```diff
@@ -252,6 +266,28 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: optional-operators-ci-operator-sdk-hypershift
+- as: e2e-aws-single-node-integration-tests
```

Comment:
I would instead name this `e2e-aws-single-node-validation-tests` so it doesn't get confused with our already existing integration test pipeline